core/clients/types/endpoint.go

```diff
@@ -11,16 +11,13 @@ import (
 type Endpoint struct{}
 
 func (e Endpoint) Monitor(params EndpointParams, ch chan string) {
-	check := time.Now()
 	for {
-		if time.Now().After(check) {
-			data, err := consulclient.GetServiceEndpoint(params.ServiceKey)
-			if err != nil {
-				fmt.Fprintln(os.Stdout, err.Error())
-			}
-			url := fmt.Sprintf("http://%s:%v%s", data.Address, data.Port, params.Path)
-			ch <- url
-			check = time.Now().Add(time.Second * time.Duration(15))
+		data, err := consulclient.GetServiceEndpoint(params.ServiceKey)
+		if err != nil {
+			fmt.Fprintln(os.Stdout, err.Error())
 		}
+		url := fmt.Sprintf("http://%s:%v%s", data.Address, data.Port, params.Path)
+		ch <- url
+		time.Sleep(time.Second * time.Duration(15))
```

Comment:
Please do not hard code magic numbers. Always declare a constant.

Reply:
@drasko This will be a configurable entry for Delhi.